css/properties/font-style.json

```
@@ -132,7 +132,9 @@
             ],
             "support": {
               "chrome": {
-                "version_added": "62"
+                "version_added": "62",
+                "partial_implementation": true,
+                "notes": "The provided angle has little effect. Generally, angles from 14 to 90 are shown with a slant of 14 degrees, like `font-style: oblique` without a provided angle, and angles from -90 to 13 are generally shown without slant, like `font-style: normal`. In some situations, like with a vertical `writing-mode` being used, a negative angle does produce a slant. See [bug 425388876](https://issues.chromium.org/issues/425388876)"
```

Let's rephrase to typical BCD notes style:

```suggestion
                "notes": "Angles from 14 to 90 have the same effect as `font-style: oblique` (with a slant of 14 degrees). Angles from from -90 to 13 have the same effect as `font-style: normal` (without a slant). In some cases (e.g. with vertical `writing-mode`), a negative angle produces a slant. See [bug 425388876](https://issues.chromium.org/issues/425388876)."
```